src/components/auth0/Auth.js

```diff
@@ -0,0 +1,107 @@
+import auth0 from 'auth0-js';
```

Comment:
this is not a react component and thus does not belong in the `components` folder. move it to `src/auth0/Auth.js`